content/en/checkout/demo.md

```diff
@@ -5,4 +5,6 @@ position: 31
 category: Checkout
 ---
 
-Conteúdo
```

Comment:
You could add a title here like `Live Integration Demo`

Reply:
The title is automatically inserted by nuxt-docs, with the same name as the menu entry.
![Screenshot 2022-07-08 at 10 26 38](https://user-images.githubusercontent.com/2780456/177962273-17f5ee5a-fb99-41a6-bdcd-84e059dcd748.png)

Reply:
Didn't know. Looks good.